hysr/types.py

```diff
@@ -0,0 +1,17 @@
+import typing
+
+""" For functions accepting either an int or a list of int as arguments"""
+ListOrIndex = typing.Union[int, typing.Sequence[int]]
```

Comment:
I think to be connected correctly docstrings for variables would need to be defined below the variable.  Alternatively, you can use comments starting with `#:` before the variable.

Reply:
typing now provides "Annotated", but starting only from python3.9:
 https://stackoverflow.com/questions/8820276/docstring-for-variable/8820636